src/util/Collection.js

```diff
@@ -60,13 +60,13 @@ class Collection extends Map {
 
   /**
    * Obtains the first item in this collection.
-   * @param {number} [count] Number of values to return. If omited, returns a single values.
+   * @param {number} [count] Number of values to return. If omited, returns a single value.
    * If present, returns an array of values.
    * @returns {*|Array}
    */
   first(count) {
     if (!count) return this.values().next().value;
```

Comment:
I actually thought of doing that, but I decided against it, because you might have an "arbitrary" count and then you need to make sure the return is consistently an array regardless of the number